internal/controllers/controller_test.go

```diff
@@ -109,3 +109,136 @@ func Test_PodReconcile(t *testing.T) {
 		})
 	}
 }
+
+func Test_areImagesChanged(t *testing.T) {
+	type args struct {
+		oldPod *corev1.Pod
+		newPod *corev1.Pod
+	}
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "the same images in different order",
+			args: args{
```

Comment:
Can we shorten test cases to something like:
```
newImages: {
initContainers: {}
Containers:{}
}
```
and create those pods in test run?